extension/manifest.json

```diff
@@ -15,5 +15,9 @@
   "browser_action": {
     "default_popup": "popup.html"
   },
-  "permissions": ["storage"]
+  "permissions": ["storage"],
+  "web_accessible_resources": [
+    "style.css"
+  ],
+  "content_security_policy": "worker-src github.com/assets-cdn/worker/ gist.github.com/assets-cdn/worker/ blob:;"
```

Comment:
added this line because I was getting issue below.
```
Refused to create a worker from 'blob:https://github.com/<uuid>' because it violates the following Content Security Policy directive: "worker-src github.com/assets-cdn/worker/ gist.github.com/assets-cdn/worker/"
```

Reply:
I'm still getting this error with the extension disabled so don't think this is related to us.

Reply:
you are right I just removed it now it's just css.